src/NLog.Web.AspNetCore/LayoutRenderers/AspNetResponseTrailersLayoutRenderer.cs

```diff
@@ -0,0 +1,87 @@
+﻿#if ASP_NET_CORE3
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
+using NLog.LayoutRenderers;
+using NLog.Web.Internal;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// ASP.NET Response Trailers
+    /// </summary>
+    /// <example>
+    /// <para>Example usage of ${aspnet-response-trailers}</para>
+    /// <code lang="NLog Layout Renderer">
+    /// ${aspnet-response-trailers:OutputFormat=Flat}
+    /// ${aspnet-response-trailers:OutputFormat=JsonArray}
+    /// ${aspnet-response-trailers:OutputFormat=JsonDictionary}
+    /// ${aspnet-response-trailers:OutputFormat=JsonDictionary:TrailerNames=username}
+    /// ${aspnet-response-trailers:OutputFormat=JsonDictionary:Exclude=access_token}
+    /// </code>
+    /// </example>
+    [LayoutRenderer("aspnet-response-trailers")]
+    public class AspNetResponseTrailersLayoutRenderer : AspNetLayoutMultiValueRendererBase
+    {
+        /// <summary>
+        /// Trailer names to be rendered.
+        /// If <c>null</c> or empty array, all trailers will be rendered.
+        /// </summary>
+        public List<string> TrailerNames { get; set; }
+
+        /// <summary>
+        /// Gets or sets the keys to exclude from the output. If omitted, none are excluded.
+        /// </summary>
+        /// <docgen category='Rendering Options' order='10' />
+        public ISet<string> Exclude { get; set; } =
+            new HashSet<string>(new[] { "ALL_HTTP", "ALL_RAW", "AUTH_PASSWORD" }, StringComparer.OrdinalIgnoreCase);
+
+        /// <summary>
```

Comment:
Can this be replaced by  `<inheritdoc/>` ?

Reply:
Complete